spacy/lang/hi/lex_attrs.py

```diff
@@ -40,13 +41,85 @@
     "अठारह",
     "उन्नीस",
     "बीस",
+    "इकीस", "इक्कीस",
+    "बाईस",
+    "तेइस",
+    "चौबीस",
+    "पच्चीस",
+    "छब्बीस",
+    "सताइस", "सत्ताइस",
+    "अट्ठाइस",
+    "उनतीस",
     "तीस",
+    "इकतीस",
+    "बतीस", "बत्तीस",
```

Comment:
'त्तीस' is needed for each of thirties numbers

Reply:
Good Catch. Done!